bundles/http/Citrix/Microsoft-Outlook/script.js

```diff
@@ -453,12 +101,57 @@ async function myEvents(client, dataStore, userIds) {
 }
 
 
-
-
 async function editRecurringEventwithCurrentTimezone(param) {
     const { client, dataStore, actionParameters } = param
-    const start = moment().utc().subtract(1, 'm').format()
+    const start = moment.utc().subtract(1, "m").format()
     const end = moment().utc().add(30, 'd').format()
+    const attendees = [
+        {
+            "emailAddress": {
+                "address": `${actionParameters?.email1}`
+            },
+            "type": `${actionParameters.type1}`
+        }
+    ]
+    if (actionParameters?.email2) {
+        attendees.push(
+            {
+                "emailAddress": {
+                    "address": `${actionParameters.email2}`
+                },
+                "type": `${actionParameters.type2}`
+            }
+        )
+    } else if (actionParameters.email3) {
+        attendees.push({
+            "emailAddress": {
+                "address": `${actionParameters.email3}`
+            },
+            "type": `${actionParameters.type3}`
+        })
+    }
```

Comment:
This code will not work as expected. Test this attendee scenario and modify the code accordingly.